test/javascripts/acceptance/chat-flagging-test.js

```diff
@@ -29,6 +29,7 @@ const chatSettled = async () => {
 };
 
 acceptance("Discourse Chat - Flagging test", function (needs) {
+  let defaultChatView;
```

Comment:
Should we reset this variable after each test?

Reply:
I think it's fine as is, it's kind of a weird reason we use this (because we are accessing the chat message IDs from the view) in this test, and it's only used within this acceptance function. The hoops we have to jump through because we don't have let/context in JS! :laughing:

Reply:
Ooo icic. I wasn't sure if `generateChatView` would be returning different ids each time. All good here :)